traveler-ios-ui/TravelerKitUI/ViewControllers/CancellationViewController.swift

```diff
@@ -68,6 +145,14 @@ class CancellationViewController: UITableViewController {
     }
 }
 
+extension CancellationViewController: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        view.endEditing(true)
+        explanation = (textField.text?.isEmpty ?? true) ? nil : textField.text
```

Comment:
This line is not working properly. When writing an explanation and cancelling my request, the payload does not send the text in that field.

Reply:
I have the explanation text in my JSON body. Could you please provide more details?